lib/rules/no-route-action.js

```diff
@@ -25,12 +23,26 @@ export default class NoRouteAction extends Rule {
     if (maybeAction !== 'route-action') {
       return;
     }
+
     if (node.path.data === true || node.path.this === true) {
       return;
     }
+
+    const routeActionName = node.params[0].value;
+
     this.log({
-      message: ERROR_MESSAGE.replace('%', usageContext),
+      message: makeErrorMessage(routeActionName, isSubExpression),
       node,
     });
   }
 }
+
+function makeErrorMessage(actionName, isSubExpression) {
+  const PLACE_HOLDER = '%';
+  const ERROR_MESSAGE = `Do not use \`route-action\` as ${PLACE_HOLDER}. Instead, use controller actions.`;
+  const usageContext = isSubExpression
+    ? `(route-action '${actionName}')`
+    : `{{route-action '${actionName}'}}`;
+
+  return ERROR_MESSAGE.replace(PLACE_HOLDER, usageContext);
```

Comment:
Can we eliminate this replacement? I don't see why there should be any replacements/placeholders involved.